core/trino-main/src/main/java/io/trino/metadata/GlobalFunctionCatalog.java

```diff
@@ -54,7 +54,6 @@
 @ThreadSafe
 public class GlobalFunctionCatalog
 {
-    public static final String GLOBAL_CATALOG = "system";
```

Comment:
Can you add some details to the commit explaining how the built in functions are now available for resolution after this change? If I understand it correctly, there's no named catalog for them -- it's just a magic catalog that's always present regardless of what the PATH says, right?